components/toolbar/brave_toolbar_model_impl.cc

```diff
@@ -18,11 +20,14 @@ const base::string16 original_scheme_part =
 const base::string16 replacement_scheme_part =
                                   base::ASCIIToUTF16(kInternalUIScheme);
 
+const base::string16 pdfjs_url_prefix =
```

Comment:
can we use BraveContentBrowserClient::BrowserURLHandlerCreated to add a HandlerPair for this? That's typically how it would work